templates/login.html

```diff
@@ -49,5 +49,12 @@ <h1 class="text-2xl font-bold tracking-tight text-center">
       ></span>
       Sign in
     </button>
+    <button
+      type="submit"
+      class="leading-6 bg-neutral-300 mt-5 font-semibold rounded py-1.5 flex items-center justify-center disabled:bg-neutral-500 text-neutral-900"
+      onclick="window.location.href='/login_google'; return false;"
```

Comment:
Can we use an `<a href="/login_google">` element here? It's better for accessibility.